diff --git a/src/main/java/com/cybage/controller/HomeController.java b/src/main/java/com/cybage/controller/HomeController.java
index 996b9a2..a19cf74 100644
--- a/src/main/java/com/cybage/controller/HomeController.java
+++ b/src/main/java/com/cybage/controller/HomeController.java
@@ -9,5 +9,6 @@ public class HomeController {
   {
 
       System.out.println("hello samiksha");
+      System.out.println("hello samiksha");
   }
 }
